src/systemverilog/systemverilog.ts

```diff
@@ -163,11 +163,12 @@ export const language = <ILanguage>{
 			[/\/\/.*$/, 'comment'],
 		],
 
-		comment: [
-			[/[^\/*]+/, 'comment'],
-			[/\*\//, 'comment', '@pop'],
-			[/[\/*]/, 'comment']
+	    comment: [
+				[/[^\/*]+/, 'comment'],
+				[/\*\//, 'comment', '@pop'],
+				[/[\/*]/, 'comment']
 		],
+
 		//Identical copy of comment above, except for the addition of .doc
 		doccomment: [
```

Comment:
did you look into what doccoment does and whether or not we need it?

Reply:
removed